### DISCUSSION TITLE
Please add Sweden, Borås

### DISCUSSION THREAD

**Discussion by ghost** [mampfes/hacs_waste_collection_schedule#2329] [Ideas]
I use another, very unsupported integration for this where I hacked in support for my area.

Country : Sweden
Area : Borås
Company: Borås Energi och miljö 
Api path / syntax: https://kundportal.borasem.se/EDPFutureWeb/SimpleWastePickup/GetWastePickupSchedule?address={{ 'Streetname 1, City (1234)' | urlencode()}}}}
Path to lookup page : https://www.borasem.se/webb/privat/avfallochatervinning/abonnemangforhushallsavfall/nastatomningsdag.4.5a231a8f188bd840a1327da.html

Test addresses :
Länghemsgatan 10, Borås (217743)
Yttre Näs 1, Seglora (224837)
Stora Hyberg 1, Brämhult (213836)

Entities:
To my knowledge Borås Energi och miljö offers 3 different data types.
1+2 are waste bins with different departments for different types of waste eg. Organic, plastic packages, paper packages...
The entitie names for waste is "kärl 1" & "kärl 2".
There is a 3rd data type and that is septic tank. That would also be good if it's possible to support. 3rd address example contains this 3rd data type named "slam"

If more data needed, I will try to help

**Comment by 5ila5** [comment]
moved to #2333

**Comment by 5ila5** [comment]
Borås is supported by the [edpevent_se source](https://github.com/morotsgurka/hacs_waste_collection_schedule/blob/master/doc/source/edpevent_se.md), you can use it with `url: https://kundportal.borasem.se/EDPFutureWeb/SimpleWastePickup` (you will be able to use `service_provider: boras` instead after the next update)
